src/game/Tactical/PathAI.cc

```diff
@@ -1735,13 +1730,13 @@ INT32 FindBestPath(SOLDIERTYPE* s, INT16 sDestination, INT8 ubLevel, INT16 usMov
 						SLOGD(zTempString );
```

Comment:
This no longer compiles because SLOGD expects a char * as its first argument. When was the last time anyone used PATHAI_SKIPLIST_DEBUG anyway?

Reply:
Doesn't hurt to fix it; unlike the video debugging, this might still be useful.